static/components/players-widgets.js

```diff
@@ -101,7 +101,7 @@ class PlayersWidgets extends LitElement {
                         : html`
                             <div class="player-info>
                                 <p class="player-name"><span class="highlight">Passing Leader:</span> ${this.players.passing.first_name} ${this.players.passing.last_name}</p>
-                                <p class="player-stats"><span class="highlight">Passing Yards:</span> ${this.players.passing.passing_yards} yards</p>
+                                <p class="player-stats"><span class="highlight">Passing Yards:</span> ${Math.round(this.players.passing.passing_yards)} yards</p>
```

Comment:
Try doing this on the backend instead. This way, all calculation and data will be handled on the backend side of things.